android/modules/ui/src/java/ti/modules/titanium/ui/widget/webview/TiUIWebView.java

```diff
@@ -504,6 +515,33 @@ public void propertyChanged(String key, Object oldValue, Object newValue, KrollP
 		}
 	}
 
+	private void zoomBy(WebView webView, float scale)
+	{
+		if (webView != null) {
+			if (scale < 0.0f) {
+				scale = 0.0f;
+			} else if (scale >= 100.0f) {
+				scale = 100.0f;
+			}
+			webView.zoomBy(scale);
+		}
+	}
+
+	public void zoomBy(float scale)
+	{
+		zoomBy(getWebView(), scale);
```

Comment:
This public zoomBy() method is missing an API Level 21 check like how we handle the zoom level property. Either add the API Level check here too or move all the API Level checks to the private zoomBy() method.